src/main/resources/chains/mordor-chain.conf

```diff
@@ -87,6 +87,11 @@
   # https://ecips.ethereumclassic.org/ECIPs/ecip-1097
   ecip1097-block-number = "1000000000000000000"
 
+
+  # Epoch calibration block number
+  # https://ecips.ethereumclassic.org/ECIPs/ecip-1099
+  ecip1099-block-number = "2520000"
```

Comment:
this has already been reached on Mordor, right? Have we synced and try it there already?

Reply:
Good point. I will sync with the Mordor before the merge